_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,72 @@
+
+## OpenSearch is for everyone, and we are inclusive
+
+We’d like to begin this ongoing conversation with a subject that is foundational to who we are and how we talk to each other as a community: inclusion. As an open-source project, OpenSearch is for everyone, and we are inclusive. We value the diversity of backgrounds and perspectives in the OpenSearch community and welcome feedback from any contributor, regardless of their experience level. 
+
+When developing OpenSearch documentation, we strive to create content that is inclusive and free of bias. We use inclusive language to connect with the diverse and global OpenSearch audience, and we are careful in our word choices. Inclusive and bias-free content improves clarity and accessibility of our content for all audiences, so we avoid ableist and sexist language and language that perpetuates racist structures or stereotypes. In practical terms, this means that we do not allow certain terms to appear in our content, and we avoid using others, depending on the context.
+
+## Inclusive terminology
+
+The following tables list offensive terms that we do not use in our content and sensitive terms that we avoid using. 
+
+We do not use the following terms in OpenSearch content. Instead, use one of the acceptable alternatives provided in the second column.
+
+| Don’t use      | Use instead                 |
+|----------------|-----------------------------|
+| abort          | stop                        |
+| black day      | blocked day                 |
+| blacklist      | deny list                   |
+| execute        | start, run                  |
+| hang           | stop responding             |
+| kill           | end, stop                   |
+| master         | primary, main, leader       |
+| master account | management account          |
+| slave          | replica, secondary, standby |
+| white day      | open day                    |
+| whitelist      | allow list                  |
+
+We avoid using the following terms in OpenSearch content, depending on the context. Again, you can use one of the alternatives listed in the second column.
+
```

Comment:
Minor edit - please insert this above the table:

`<div class="table-styler"></div>`

thanks